src/cljs/main/broadfcui/page/billing/billing_management.cljs

```diff
@@ -69,64 +70,63 @@
        (:error-message @state) (style/create-server-error-message (:error-message @state))
        (nil? (:projects @state)) [comps/Spinner {:text "Loading billing projects..."}]
        :else
-       [table/Table
-        {:reorderable-columns? false
-         :header-row-style table-style/header-row-style-light
-         :row-style table-style/table-row-style-light
-         :resize-tab-color (:line-default style/colors)
-         :columns [{:starting-width 32 :resizable? false
-                    :sort-by :none
-                    :content-renderer
-                    (fn [creationStatus]
-                      [:span {:title creationStatus}
-                       (moncommon/icon-for-project-status creationStatus)])}
-                   {:header "Project Name" :starting-width 500
-                    :as-text #(% "projectName") :sort-by :text
-                    :sort-initial :asc
-                    :content-renderer
-                    (fn [{:strs [projectName role creationStatus message]}]
-                      [:span {}
-                       (cond
-                         (= creationStatus project-status-creating)
-                         [PendingProjectControl
-                          {:project-name projectName
-                           :on-status-change (partial this :-handle-status-change projectName)}]
-                         (and (= creationStatus project-status-ready) (= role "Owner"))
-                         (style/create-link {:text projectName
-                                             :onClick #((:on-select props) projectName)})
-                         :else projectName)
-                       (when message
-                         [:div {:style {:float "right" :position "relative"}}
-                          [common/FoundationInfoBox
-                           {:text [:div {} [:strong {} "Message:"] [:br] message]}]])])}
-                   {:header "Role" :starting-width :remaining :resizable? false}]
+       [Table
+        {:table {:behavior {:reorderable-columns? false}
```

Comment:
Let's rename this key to `:core` or `:body` since we've established "table" to mean the entire thing including toolbars and pagination.